examples/vnet/doca_dpdk.c

```diff
@@ -1496,6 +1509,22 @@ doca_dpdk_add_pipe_entry(struct doca_flow_pipe *pipe,
 	return NULL;
 }
 
+int doca_dpdk_find_pipe_entry(uint16_t pipe_queue,
+			      struct doca_flow_pipe *pipe,
+			      struct doca_flow_pipe_entry *pipe_entry)
+{
+	struct doca_flow_pipe_entry *entry;
+	int found = 0;
+
+	LIST_FOREACH(entry, &pipe->entry_list[pipe_queue], next) {
```

Comment:
we should never use linked-list for more than few objects.
see my comment below, you don't need to find it.

Reply:
OK, Why we need 'pipe_queue' in doca_flow_pipe_rm_entry() if not need search?
Only need remove the entry resource is enough.